### DISCUSSION TITLE
About scrollcontainer  , The problem is shown in the video

### DISCUSSION THREAD

**Discussion by YTLoudYN** [beeware/toga#1912] [Q&A]
https://user-images.githubusercontent.com/131869902/235289945-4317f648-4e9f-4f9c-9bee-060547653160.mp4

'''
import toga
from toga.constants import COLUMN, ROW, RED
from toga.style import Pack

class TEST(toga.App):
    TOGGLE_CHUNK = 10

    vscrolling = True
    hscrolling = False
    scroller = None

    def startup(self):

        self.main_box = toga.Box(style=Pack(direction=COLUMN))

        self.box = toga.Box()
        self.box.style.direction = COLUMN
        self.box.style.padding = 10
        self.scroller = toga.ScrollContainer(
            horizontal=self.hscrolling, vertical=self.vscrolling
        )

        self.box2 = toga.Box()
        self.box2.add(
            toga.Label('Thie is a TEST( Print 0 ~ 100 )'),
            toga.Button('START',on_press=self.start_on)
        )
        self.box.add(self.box2)

        self.scroller.content = self.box

        self.main_window = toga.MainWindow(title=self.formal_name)
        self.main_window.content = self.scroller
        self.main_window.show()

    def start_on(self,args):

        self.box.remove(self.box2)

        self.box2 = toga.Box(style=Pack(direction=COLUMN))

        for i in range(100):
            self.box2.add(
                toga.Label('>>>' + str(i))
            )

        self.box.add(
            toga.Label('Print 0 ~ 100', style=Pack(color=RED)),
            self.box2
        )
        self.main_window.show()

def main():
    return TEST('TEST','id')

if __name__ == '__main__':
    main().main_loop()
'''

**Comment by freakboy3742** [comment]
Looks like you've found a bug in ScrollContainer. If I had to guess, I'd say that the scroller isn't responding to dynamic changes in the inner content.

**Reply by YTLoudYN** [reply]
https://user-images.githubusercontent.com/131869902/235331233-77908ebb-62d2-43c2-99ad-b9128c845a35.mp4
Rolling container example run: video above


This is the same problem as the scrollcontainer in the example, please ask, is there any solution?

**Reply by freakboy3742** [reply]
You've found a bug. If I had a workaround, I'd provide one. If this is affecting you, I'd suggest investigating the internals and trying to debug the problem.
